x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.test.tsx

```diff
@@ -87,6 +86,10 @@ const defaultProps = {
 };
 
 describe('Actions', () => {
+  beforeAll(() => {
+    (useShallowEqualSelector as jest.Mock).mockReturnValue(mockTimelineModel);
```

Comment:
I don't know why this problem was happening, but I had to change the way the `useShallowEqualSelector` mock and set it here instead of providing `jest.mock()` a factory function. The error I was getting (including in CI) was:

```
TypeError: Cannot read properties of undefined (reading 'mockTimelineModel')

      24 | }));
      25 | jest.mock('../../../../../common/hooks/use_selector', () => ({
    > 26 |   useShallowEqualSelector: jest.fn().mockReturnValue(mockTimelineModel),
         |                                                      ^
      27 | }));
      28 | jest.mock(
      29 |   '../../../../../detections/components/alerts_table/timeline_actions/use_investigate_in_timeline',

      at x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.test.tsx:26:54
```